src/main/java/com/conveyal/datatools/manager/auth/Auth0UserProfile.java

```diff
@@ -64,15 +60,28 @@ public static Auth0UserProfile createSystemUser() {
             }
         );
         adminDatatoolsInfo.setClientId(DataManager.getConfigPropertyAsText("AUTH0_CLIENT_ID"));
-
-        Auth0UserProfile.AppMetadata adminAppMetaData = new Auth0UserProfile.AppMetadata();
-        adminAppMetaData.setDatatoolsInfo(adminDatatoolsInfo);
-
+        Auth0UserProfile.AppMetadata adminAppMetaData = createDefaultAppMetaData();
         Auth0UserProfile adminUser = new Auth0UserProfile("system", "user_id:system");
         adminUser.setApp_metadata(adminAppMetaData);
         return adminUser;
     }
 
+    /**
+     * Create app meta data to be used by methods constructing an {@link Auth0UserProfile}.
+     */
+    private static Auth0UserProfile.AppMetadata createDefaultAppMetaData() {
```

Comment:
This method name and javadoc should clarify that it is ONLY for application admins' appMetaData.

Reply:
@landonreed I should have reverted the changes made here. I was originally attempting to create a user with the required app meta data, but realized it would be far easier to go to Auth0 (and update the app meta data there). However, I have updated inline with your suggestion.